drivers/i2c/i2c_mchp_xec.c

```diff
@@ -10,10 +10,13 @@
 #include <kernel.h>
 #include <soc.h>
 #include <errno.h>
+#include <drivers/gpio.h>
 #include <drivers/i2c.h>
 #include <logging/log.h>
 LOG_MODULE_REGISTER(i2c_mchp, CONFIG_I2C_LOG_LEVEL);
 
+/* #define DEBUG_I2C_XEC_ITM */
```

Comment:
Remove or document usage

Reply:
My bad. Missed this line as I removed all my other custom ITM debug code.